evm/foundry.toml

```diff
@@ -8,6 +8,16 @@ src = "src"
 out = "out"
 libs = ["lib"]
 
+[profile.prod]
+solc_version = "0.8.19"
+optimizer = true
+optimizer_runs = 200
+via-ir = true
```

Comment:
I think we should keep just this line -- the rest are inherited from the default profile